pkg/venafi/tpp/tpp.go

```diff
@@ -63,28 +63,28 @@ type device struct {
 }
 
 type certificateRequest struct {
-	PolicyDN                string          `json:",omitempty"`
-	CADN                    string          `json:",omitempty"`
-	ObjectName              string          `json:",omitempty"`
-	Subject                 string          `json:",omitempty"`
-	OrganizationalUnit      string          `json:",omitempty"`
-	Organization            string          `json:",omitempty"`
-	City                    string          `json:",omitempty"`
-	State                   string          `json:",omitempty"`
-	Country                 string          `json:",omitempty"`
-	SubjectAltNames         []sanItem       `json:",omitempty"`
-	Contact                 string          `json:",omitempty"`
```

Comment:
For some reason, the `Contact` field here was misspelled (should be `Contacts`) and it type was incorrect: it should be of the form `[{"PrefixedUniversal": "..."}]`.